client/src/components/survey/SurveyPage3.jsx

```diff
@@ -104,8 +104,7 @@ const mapDispatchToProps = dispatch => {
   return {
     setSurveyPage3: survey => dispatch(actions.setSurveyPage3(survey)),
     toSurveyPage2: survey => dispatch(actions.toSurveyPage2(survey)),
-    submitSurvey: (fileNumber, observation, encryptOptions, userSession) =>
-      dispatch(actions.submitSurveyThunk(fileNumber, observation, encryptOptions, userSession)),
+    addObservation: (userSession, survey) => dispatch(actions.addObservation(userSession, survey)),
```

Comment:
Unless I'm missing something, it looks like you changed the function that adds an observation from a thunk to a typical action - shouldn't we keep this as a thunk?

Reply:
It's a thunk.

Reply:
Whoops, sorry about that